subjects/devops/concat_string/README.md

```diff
@@ -0,0 +1,31 @@
+## concat_string
+
+### Instructions
+
+Create a file `concat_string.py` that contains a function `concat` which takes in two strings as parameters and performs the following operations:
+
+- Concatenates the two strings by adding a comma and space between them.
+- Returns the resulting string.
+
+### Usage
+
+Here is an example of how to use the `concat` function:
+
+```python
+import concat_string
+string1 = "Hello"
+string2 = "World"
+print(concat_string.concat(string1, string2))
+```
+
+This will output:
+
+```console
+$ python3 test.py
+Hello World
```

Comment:
According to the subject and solution provided, the output should be:

```console
Hello, World
```

Reply:
Good catch, thanks.